pd-recover.md

```diff
@@ -6,145 +6,120 @@ aliases: ['/docs/dev/pd-recover/','/docs/dev/reference/tools/pd-recover/']
 
 # PD Recover User Guide
 
-PD Recover is a disaster recovery tool of PD, used to recover the PD cluster which cannot start or provide services normally. PD Recover is downloaded with TiDB Ansible in the `resource/bin/pd-recover` path.
+PD Recover is a disaster recovery tool of PD, used to recover the PD cluster which cannot start or provide services normally.
+
+## Compile from source code
+
+1. [Go](https://golang.org/) Version 1.13 or later because the Go modules are used.
+2. In the root directory of the [PD project](https://github.com/pingcap/pd), use the `make pd-recover` command to compile and generate `bin/pd-recover`
+
+> **Note:**
+>
+> Generally, you don't need to compile source code as the PD Control tool already exists in the released Binary or Docker. However, dev users can refer to the above instruction for compiling source code.
+
+## Download TiDB installation package
+
+If you want to download the latest version of `pd-recover`, directly download the TiDB package, because `pd-recover` is included in the TiDB package.
+| Package download link | OS | Architecture | SHA256 checksum |
```

Comment:
```suggestion
| Package name | OS | Architecture | SHA256 checksum |
```

Reply:
I think the download link is better?

Reply:
I agree with you, but the [download](https://docs.pingcap.com/tidb/stable/download-ecosystem-tools#download) page on PingCAP website uses `Package name`. For consistency, I think it would be better to use `Package name`.
![image](https://user-images.githubusercontent.com/41534398/87656116-a6244d80-c78b-11ea-977d-0c3a88742d08.png)

Reply:
updated